miden-lib/src/transaction/account_stub.rs

```diff
@@ -3,11 +3,21 @@ use crate::memory::{
     ACCT_NONCE_IDX, ACCT_STORAGE_ROOT_OFFSET, ACCT_VAULT_ROOT_OFFSET,
 };
 use miden_objects::{
-    accounts::{Account, AccountId, AccountStorage, AccountStorageDelta, AccountStub},
-    crypto::merkle::{merkle_tree_delta, MerkleStore, MerkleStoreDelta, NodeIndex},
+    accounts::{
+        Account, AccountId, AccountStorage, AccountStorageDelta, AccountStub, AccountVaultDelta,
+    },
+    assets::Asset,
+    crypto::{
+        hash::rpo::RpoDigest,
```

Comment:
Instead of importing `RpoDigest` we can import `Digest` from `miden_objects` - they are the same thing. This whole import could look like:
```Rust
use miden_objects::{
    accounts::{
        Account, AccountId, AccountStorage, AccountStorageDelta, AccountStub, AccountVaultDelta,
    },
    assets::Asset,
    crypto::merkle::{merkle_tree_delta, MerkleStore, MerkleStoreDelta, NodeIndex},
    transaction::FinalAccountStub,
    utils::{
        collections::{BTreeMap, Diff},
        vec,
        vec::Vec,
    },
    AccountError, Digest, Felt, TransactionResultError, Word,
};
```